src/components/grid-aware/BlockQuoteBlock/BlockQuoteBlock.stories.js

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import Slider from "react-slick";
+import BlockQuoteBlock from "./BlockQuoteBlock";
+/* eslint-disable react/jsx-props-no-spreading */
+
+import "slick-carousel/slick/slick.css";
+import "slick-carousel/slick/slick-theme.css";
+
+export default {
+  title: "Grid-Aware/BlockQuoteBlock",
+  component: BlockQuoteBlock,
+};
+
+const settings = {
+  dots: true,
+  infinite: true,
+  speed: 500,
+  slidesToShow: 1,
+  slidesToScroll: 1,
+};
+
+const Template = ({ quote, author }) => (
+  <Slider {...settings}>
```

Comment:
This really ought to be part of our `<BlockQuoteBlock>` component itself. Otherwise, you have to instantiate your own carousel every time you want to use it. I think you should instead make the component take in an array of quote/attribution objects.

Reply:
Ok, I took the carousel and made it part of the blockquoteblock component instead. Will you please take a look at the array I built? Is that how you want it?

Reply:
Thanks for moving the carousel into the component itself, but I have a few other comments about the component now, which I'll leave as new inline comments.